docs/Installation.rst

```diff
@@ -53,8 +55,9 @@ On Ubuntu 14.10, one will need to run the following script to satisfy all depend
 
 .. sourcecode:: bash
 
-    sudo apt-get install build-essential fpc postgresql postgresql-client \
-         gettext python2.7 iso-codes shared-mime-info stl-manual cgroup-lite
+    sudo apt-get install build-essential openjdk-8-jre openjdk-8-jdk fpc \
```

Comment:
For what it's worth:

> OpenJDK is the official reference implementation of Java SE since version 7.

So I would not include it... In some cases it may be useful to include different implementations (e.g. pypy) but in this case I don't see the point... Also, if I'm not mistaken, it's quite hard to install Oracle JDK >=7 on Ubuntu (you may need to use an external PPA)